packages/ecs-agent/0001-bottlerocket-default-filesystem-locations.patch

```diff
@@ -41,37 +24,11 @@ index 38f0d0f..acebb26 100644
  
  	// DefaultTaskMetadataSteadyStateRate is set as 40. This is arrived from our benchmarking
  	// results where task endpoint can handle 4000 rps effectively. Here, 100 containers
-diff --git a/agent/config/config_unix.go b/agent/config/config_unix.go
-index 6c1254d..3adefaa 100644
---- a/agent/config/config_unix.go
-+++ b/agent/config/config_unix.go
-@@ -29,10 +29,10 @@ const (
- 	// AgentCredentialsAddress is used to serve the credentials for tasks.
- 	AgentCredentialsAddress = "" // this is left blank right now for net=bridge
- 	// defaultAuditLogFile specifies the default audit log filename
--	defaultCredentialsAuditLogFile = "/log/audit.log"
-+	defaultCredentialsAuditLogFile = "/var/log/ecs/audit.log"
- 
- 	// defaultRuntimeStatsLogFile stores the path where the golang runtime stats are periodically logged
--	defaultRuntimeStatsLogFile = `/log/agent-runtime-stats.log`
-+	defaultRuntimeStatsLogFile = `/var/log/ecs/agent-runtime-stats.log`
- 
- 	// DefaultTaskCgroupV1Prefix is default cgroup v1 prefix for ECS tasks
- 	DefaultTaskCgroupV1Prefix = "/ecs"
-@@ -63,7 +63,7 @@ func DefaultConfig() Config {
- 		DockerEndpoint:                      "unix:///var/run/docker.sock",
- 		ReservedPorts:                       []uint16{SSHPort, DockerReservedPort, DockerReservedSSLPort, AgentIntrospectionPort, AgentCredentialsPort},
- 		ReservedPortsUDP:                    []uint16{},
--		DataDir:                             "/data/",
-+		DataDir:                             "/var/lib/ecs/data",
- 		DataDirOnHost:                       "/var/lib/ecs",
- 		DisableMetrics:                      BooleanDefaultFalse{Value: ExplicitlyDisabled},
- 		ReservedMemory:                      0,
```

Comment:
It looks like the agent supports these things now as settings and we are adding them to our default configuration. Is this patch still needed? It looks like it no longer is about supporting paths but maybe fixing an attribution file?

Reply:
I have deleted these and added them as settings. I believe github highlight it because of + sign. It is not in final file